config/Sage/maps.txt

```diff
@@ -50,5 +50,4 @@ endmap
 
 map kilostation
 	maxplayers 60
```

Comment:
```suggestion
	maxplayers 60
	disabled
```

`votable` is technically fine to remain, but you need to define the map as disabled for it to be removed from rotation.